app/src/main/java/org/fossasia/openevent/app/common/BaseFragment.java

```diff
@@ -1,13 +1,33 @@
 package org.fossasia.openevent.app.common;
 
+import android.app.Activity;
 import android.support.v4.app.Fragment;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
 
 import com.squareup.leakcanary.RefWatcher;
 
 import org.fossasia.openevent.app.OrgaApplication;
 
+import timber.log.Timber;
+
 public abstract class BaseFragment extends Fragment {
 
+    protected void setTitle(String title) {
```

Comment:
Don't do this as it will be problem for two pane mode. (As we don't want to change the action bar title in two pane mode)
For one pane mode, this has been already taken care at single place in `EventDetailActivity`